### DISCUSSION TITLE
How to use Javascript to focus element which is set with `useFocusRing`.

### DISCUSSION THREAD

**Discussion by jeanpan** [adobe/react-spectrum#3646] [Q&A] [answered]
As this [example](https://react-spectrum.adobe.com/react-aria/useFocusRing.html#example), I wonder if there is any way to set the focus on the button. Looks like using :focus pseudo class doesn't work either using javascript `focus()` function. 

<img width="1782" alt="Screen Shot 2022-10-14 at 16 30 11" src="https://user-images.githubusercontent.com/1307026/195788368-bdd48d6c-0b9e-4eb2-85fb-269724369210.png">

**Comment by snowystinger** [comment, marked as answer]
If you check document.activeElement, it'll be the button. The reason focus isn't visible is because focus is still in the dev console.

See https://codesandbox.io/s/objective-raman-yncg06?file=/src/App.js
Keyboard navigate to the first button, then press Enter, second button will get focus and show the focus ring

We show the focus ring only for keyboard users.

**Reply by jeanpan** [reply]
Thank you @snowystinger!
